### DISCUSSION TITLE
Wrong typo error message v1.9 from CVS server

### DISCUSSION THREAD

**Discussion by elerik** [OP-TED/eForms-SDK#834] [Q&A] [answered]
Hi labs, 

there are many error messages like this one 
BT-130-Lot is not allowed in notice type '16' under following condition: 'Type of procedure' (BT-105-Lot) is 'Open'

Problem is BT-105-Lot doesnt exist. Field is BT-105-Procedure at least in version 1.9 

Can you please take look at it and eventually fix it at CVS server. 

thanks 

Regards

**Comment by YvesJo** [comment, marked as answer]
Hi,
Thanks for pointing this out. This will get fixed.
KR
